### PR TITLE
Added file handle closing when installing a VPK to avoid locks

### PR DESCRIPTION
**Pull request by TehPsychedelic**
In all scenarios when installing a game (success, bad zip, reinstall, vitamin-dump, etc) the application will unnecessarily lock the file you installed/tried to install.

### CODE REVIEW

**Comment by Zangetsu38** [comment]
Ho good i have try understund for why get file lock after try install game

**Comment by Zangetsu38** [comment]
@TehPsychedelic if you canfind same for firmware file installed, after install is also locked

**Comment by TehPsychedelic** [comment]
@Zangetsu38 I actually checked before submitting this and didn't find any issues. What OS are you running Vita3k on?

**Comment by Zangetsu38** [comment]
ah yes, you have right, strange 🤔 just i no succes get it works for my opr, get crash like it already used 🤔

**Comment by Zangetsu38** [comment]
oups, forget, my bad, totaly forget not set one thinks for fw lol

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Vita3K is an experimental PlayStation Vita emulator for Windows, Linux and macOS
 The emulator currently runs some homebrew programs. It is also able to load some decrypted commercial games.
 
 - [Homebrew compatibility database](https://github.com/Vita3K/homebrew-compatibility/issues)
-- [Commerical compatiblity database](https://github.com/Vita3K/compatibility/issues)
+- [Commercial compatibility database](https://github.com/Vita3K/compatibility/issues)
```
Comment (VelocityRa) [inline README.md]: put this on a separate commit, has nothing to do with the main one

Comment (TehPsychedelic) [reply README.md]: Done

Comment (VelocityRa) [reply README.md]: it's still on 1 commit

Comment (1whatleytay) [reply README.md]: You can keep the change just make another commit for it. :)

Comment (TehPsychedelic) [reply README.md]: I tried the force push thing, to be honest, I hate rebases and all that, super un-intuitive on top of already un-intuitive git itself.
What you guys are saying is that this README change is still here somewhere?

Comment (VelocityRa) [reply README.md]: what @1whatleytay said

Comment (1whatleytay) [reply README.md]: No, we're saying it's gone and we like it :D 

We would rather it be in a second commit though.

Comment (TehPsychedelic) [reply README.md]: Gotcha for some reason I read a separate PR. Done.

Comment (VelocityRa) [reply README.md]: nah we're not that strict :p
